src/Compilers/CSharp/Test/Semantic/Semantics/InterpolationTests.cs

```diff
@@ -1172,9 +1172,12 @@ static void Main(string[] args)
     } 
 ";
             CreateCompilation(text).VerifyDiagnostics(
-                // (6,18): error CS8076: Missing close delimiter '}' for interpolated expression started with '{'.
+                // (6,39): error CS8089: Empty format specifier.
                 //         var x = $"{ Math.Abs(value: 1):}}";
-                Diagnostic(ErrorCode.ERR_UnclosedExpressionHole, @"""{").WithLocation(6, 18)
+                Diagnostic(ErrorCode.ERR_EmptyFormatSpecifier, ":").WithLocation(6, 39),
+                // (6,41): error CS8086: A '}' character must be escaped (by doubling) in an interpolated string.
+                //         var x = $"{ Math.Abs(value: 1):}}";
+                Diagnostic(ErrorCode.ERR_UnescapedCurly, "}").WithArguments("}").WithLocation(6, 41)
                 );
         }
```

Comment:
can we have an explicit test like these of hte case that regressed (not a parsing test, but a full compilation test).  we should also probably have the test validate the output received on multiple different framework versions.  Compiler can likely walk you through how to write those tests.

Reply:
Also need an explicit test that shows we are generating the interpolated string handler code properly.

Reply:
I created a test-theory. But I was quite a bit surprised that the test did not fail for Mscorlib40. Is the reason that the IL is identical and that the code is not executed under the target framework but the current one?